action.yml

```diff
@@ -7,6 +7,12 @@ inputs:
   protected-branches:
     description: "Comma separated list of branches that are protected."
     required: true
+  exception-branches:
+    description: "Comma separated list of branches that are excepted."
+    required: false
```

Comment:
I wonder if this could be included in the `protected-branches` option with a negation pattern. It has taken me a bit of time to remember that `exception-branches` are base branches which allow merging while `exception-prefixes` are head branches which can bypass the protection rules.

I think reducing the concepts down into two `protected-branches` and `exception-branches` would simplify this.

Proposal:

```yml
protected-branches: "main, production, !develop"
exception-prefixes: "hotfix-*,feature/*"
```

Reply:
yea, that would work! I'll get working on that! Thanks for reviewing